CDCgov/NEDSS-Modernization
# CNFT2-1184-hl7-segment-option-field

## Description

Dropdown field within the question creation page within the Messaging section (HL7 Segment)

## Tickets

* [JCNFT2-1184](https://cdc-nbs.atlassian.net/browse/CNFT2-1184)

## Steps to verify

1. If able to run UI locally, put page-builder/add/question as your URL to see the addition of the default value in the create question page.
2. Run tests that also check the status of the dropdown to verify, creation and values
